qt/lib/include/verovio-qt/verovioqttoolkit.h

```diff
@@ -0,0 +1,207 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        veroviodoc.h
+// Author:      Jonathan Schluessler
+// Created:     2017
+// Copyright (c) Authors and others. All rights reserved.
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef __VEROVIOQTTOOLKIT_H__
```

Comment:
Should be `__VRV_QT_TOOLKIT_H__`. Also, since there is the namespace it should now be just toolkit.h/cpp

Reply:
I already commented on a previous change request from you that I'd also prefer toolkit.h/cpp, but thats not possible: 

> The file has been renamed to verovioqttoolkit.h/cpp. I would have preferred to just name it toolkit.h/cpp but that's not possible because the Verovio library requires include/vrv to be in the INCLUDEPATH and thus pollutes the . include path.

Changing that would require a bigger refactoring of the Verovio library:
- Do not specify include/vrv, include/midi, ... as include directories but only include/.
- Change all .h/.cpp files to include the subdirectory of the main include. E.g. `#include "vrv/page.h"` instead of `#include "page.h"`

Reply:
I see. Since we are in the extension of the Verovio library  I would suggest qttoolkit.h/cpp. OK?

Reply:
In fact you could refactor it the other way, that is, have only `INCLUDEPATH += include/` in the Qt project (and not `include/vrvQt`) and then do `#include "vrvQt/toolkit.h". I am fine either way.

Reply:
You can have both path included, and then use give the subdirectory only for files that have a on conflict.
```
INCLUDEPATH += include/
INCLUDEPATH += include/vrvqt

{
    INCLUDEPATH += ../../include/
    INCLUDEPATH += ../../include/vrv
    INCLUDEPATH += ../../include/pugi
    INCLUDEPATH += ../../include/utf8
    INCLUDEPATH += ../../libmei
    LIBS += -L../../build -lverovio
}
```
And then to either:
```
#include "vrv/toolkit.h"
#include "vrvqt/toolkit.h"
```
For other files:
```
#include "textquickitem.h"
```